components/envs/documenter-react/jest/jest.config.js

```diff
@@ -0,0 +1,8 @@
+const reactJestConfig = require('@teambit/react/jest/jest.config');
+const { generateNodeModulesPattern } = require('@teambit/dependencies.modules.packages-excluder');
+const packagesToExclude = ['react-syntax-highlighter'];
```

Comment:
👍 
(also - we don't need the other one?)

Reply:
TODO - add a comment (I think this package had problems because it was compiled as ESM?)

Reply:
done